backend/src/main/java/gov/cdc/usds/simplereport/service/SpecimenTypeService.java

```diff
@@ -21,4 +23,14 @@ public SpecimenType fetchByInternalId(UUID internalID) {
   public List<SpecimenType> fetchSpecimenTypes() {
     return _specimenTypeRepo.findAll();
   }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public SpecimenType createSpecimenType(CreateSpecimenType input) {
```

Comment:
Would love to see some more testing for the service

Reply:
Fair enough 😄 Added a couple of tests for it.